apps/examples/wifi_manager_sample/wm_test.c

```diff
@@ -962,6 +962,9 @@ static wm_test_e _wm_get_opt(int argc, char *argv[])
 
 static int _wm_test_softap(struct options *opt, int argc, char *argv[])
 {
+	if (argc < 5) {
+		return -1;
```

Comment:
could you add log for error?

Reply:
Thank you for review. Since by right wm test will give USAGE info message if directly return, which means wrong input parameters. May I ask what log errors you want me to add?

Reply:
Let's do that by wm engineer later if it requires.